### PR TITLE
Update primary contact with new MyRack changes.

### PR DESCRIPTION
**Pull request by DonNateR**

### CODE REVIEW

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-6b50b9819e/how-to/).

**Comment by cyrichardson** [comment]
@DonNateR Suggested changes here: 
[change-to-new-primary-contact_cyrEdits2.docx](https://github.com/rackerlabs/rackspace-how-to/files/2028237/change-to-new-primary-contact_cyrEdits2.docx)

File: content/general/change-to-new-primary-contact.md [hunk 1]
```diff
@@ -5,30 +5,42 @@ title: Change an account's primary contact
...
+last_modified_date: '2018-05-22'
 last_modified_by: Nate Archer
 product: undefined
 product_url: undefined
 ---
 
-**Note:** If you are a Dedicated Hosting customer, you can request the form described in this article from your account manager.
-
-If you are the current primary contact for your account, you can update the primary contact by selecting **Account** > **Account Setting** and editing the **Primary Contact Information** section in the [Cloud Control Panel](http://mycloud.rackspace.com).
+If you are the current primary contact for your account, you can update the primary contact by selecting and editing the **Primary Contact Information** section in the [Cloud Control Panel](http://mycloud.rackspace.com), orif you're a dedicated customer, the [MyRackspace Control Panel](https://my.rackspace.com).
```
Comment (chri2547) [inline content/general/change-to-new-primary-contact.md]: orif should be 'or if'

File: content/general/change-to-new-primary-contact.md [hunk 2]
```diff
@@ -5,30 +5,42 @@ title: Change an account's primary contact
...
+#### Cloud Control Panel
+
 1. In the upper-right corner of the Cloud Control Panel, click **Account** > **Docs and Forms**.
 2. Click the **Rackspace Account Change Forms** section.
 3. From the **Change Primary Contact** section, download the **Primary Contact Change Form**.
 4. Fill out the downloaded form and save the edited file on your local machine.
 5. In the Cloud Control Panel, select **Tickets** > **Create Ticket**.
 6. In the **Category** section, set **Type** to **Account** and **Account Update**.
-7. Fill out the details for your ticket and attach your saved account change form to your ticket.
+7. Fill out the details for your ticket and attach your saved Primary Contact  Change Form to your ticket.
```
Comment (chri2547) [inline content/general/change-to-new-primary-contact.md]: Is there an extra space between 'Contact' and 'Change'?

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-7e6d3f9bd1/how-to/)

**Pull request merged**